pkg/metrics/metrics.go

```diff
@@ -70,7 +91,21 @@ func (mm *metricsManager) recordComponentVersionMetric() error {
 	return nil
 }
 
-func (mm *metricsManager) EmitGKEComponentVersion() error {
+func (mm *MetricsManager) RecordOperationErrorMetrics(
+	operationName string,
+	operationErr error,
+	diskType string) {
+	var driverName string
+	if strings.Contains(diskType, "hyperdisk") {
+		driverName = hyperdiskDriverName
+	}
+	if strings.Contains(diskType, "pd") {
+		driverName = pdcsiDriverName
+	}
+	pdcsiOperationErrorsMetric.WithLabelValues(driverName, "/csi.v1.Controller/"+operationName, LoggedError(operationErr).Error(), diskType).Set(1.0)
```

Comment:
We want the error code here not the error message. So CodeForError not LoggedError.

Reply:
Updated this to capture the error code